versioned/persist/tx/src/main/java/org/projectnessie/versioned/persist/tx/h2/H2DatabaseAdapter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.persist.tx.h2;
+
+import java.util.Arrays;
+import java.util.List;
+import org.projectnessie.versioned.persist.tx.TxDatabaseAdapter;
+import org.projectnessie.versioned.persist.tx.TxDatabaseAdapterConfig;
+
+public class H2DatabaseAdapter extends TxDatabaseAdapter {
```

Comment:
does this mean H2 is shipped as a dependency in the native image? can we control that?

Reply:
It's not in the native image (no JDBC at all yet). Wanted to leave that (add JDBC to the Quarkus image) as a separate issue/PR.